db/template.go

```diff
@@ -49,7 +49,7 @@ func (d TinkDB) CreateTemplate(ctx context.Context, name string, data string, id
 func (d TinkDB) GetTemplate(ctx context.Context, fields map[string]string) (string, string, string, error) {
 	getCondition, err := buildGetCondition(fields)
 	if err != nil {
-		return "", "", "", errors.Wrap(err, "failed to build get condition")
+		return "", "", "", errors.Wrap(err, "failed to get template")
```

Comment:
this should actually be `errors.WithMessage` not wrap but we don't need to change that now. Using errors.Wrap on an already Wrapped error will show up with a new stack trace for each `Wrap` which isn't really what we want. `WithMessage` just adds more messages to the original stack trace.